packages/salesforcedx-visualforce-markup-language-server/tsconfig.json

```diff
@@ -1,14 +1,18 @@
 {
+  "extends": "../../tsconfig.common.json",
   "compilerOptions": {
-    "module": "commonjs",
-    "target": "es6",
-    "lib": ["dom", "es6"],
-    "sourceMap": true,
-    "declaration": true,
-    "moduleResolution": "node",
     "rootDir": ".",
     "outDir": "out",
-    "preserveConstEnums": true
+    "strict": false,
+    "noImplicitAny": false,
+    // Generating Type Definitions from Javascript is blocked by requiring private name
+    // https://github.com/microsoft/TypeScript/issues/37832
+    // "allowJs": true, // for the beautify js files
```

Comment:
would we ever uncomment this?

Reply:
There are about a half-dozen failures in a couple of packages due to this check. So yes if we can clean them up we can have this enabled. Another way is to have this only in the packages that are failing and have it enabled globally.

Reply:
Created a WI and linked

Reply:
Sorry misread as question about `noUnusedLocals` in the tsconfig common json. 
For `allowJs` guess as long as we have the beautify JS src files checked in (not sure if there is a third party lib that can be imported instead that has been created since) and the TSC issue is open. This is in case someone wonders why are the JS files being copied over instead of being run through tsc. Updated comment.